testConfig.py

```diff
@@ -277,6 +347,7 @@ def parse(
             server=tuple(server),
             client=tuple(client),
             plugins=tuple(plugins),
+            simple_type=simple_type,
```

Comment:
I am a little bit confused why we need an extra state variable here.

Does this represent like arguments to a command? It might be useful to have this for other test types as well. I image maybe we need to add an extra arg to iperf for certain unique test scenarios.

Reply:
> I am a little bit confused why we need an extra state variable here.

Need is a strong word.

I thought, the use case for the simple script, is that you patch it for some testing (or future enhancements). When doing that, for testing you will necessarily need to build the test image and set `TFT_TEST_IMAGE` (to not use   `quay.io/wizhao/tft-tools:latest`). At that point, it seems maybe useful to pass additional arguments to the script.

The `simple_type` key contains test-type specific parameters for `TestType.SIMPLE`. In practice, those are extra command line arguments that are blindly passed on. Granted, that might also be useful for `TestType.IPERF_*`. But for that case, I thought we would add a `iperf_type` key (alongside `simple_type`).

Maybe extra command line arguments makes sense for all test types (in some form). So maybe the `simple_type.client_args` and `simple_type.server_args` should be hoisted to the top-level, and in some sense apply to various test types. Unclear which is preferable... (?)

Reply:
Which way do you think would be better? I am ok with hoisting it to the top-level. There was extra arg support in the old bash traffic flow test. It would definitely be nice if we prepare for something like that (not necessarily coding it)

Reply:
for arguments like `client_args`/`server_args`, that is, in an abstract sense some extra arguments we pass on (e.g. to iperf), I think it makes sense to hoist them to the higher level. That will replace `simple_type` entry.

I will do that...

Reply:
Thanks!